code/game/machinery/portable_turret.dm

```diff
@@ -717,13 +607,12 @@ Status: []<BR>"},
 		if(H.lying)
 			return
 
-	if(!emagged) // if it hasn't been emagged, it has to obey a cooldown rate
-		if(last_fired || !raised)
-			return // prevents rapid-fire shooting, unless it's been emagged
-		last_fired = 1
-		spawn()
-			sleep(shot_delay)
-			last_fired = 0
+//	if(last_fired || !raised)
```

Comment:
Because I couldn't see any difference without it, the gun still only targets and fires once each process() which seems to be 2 seconds or so, and the shot_delay is 1.5 seconds. Also it checks for !raised below that already